binds/python/bind_immutable.cpp

```diff
@@ -53,6 +53,10 @@ static void bind_immutable_module(py::module &m) {
                 return py::array(static_cast<py::ssize_t>(morpho->points().size()), morpho->points().data());
             },
             "Returns a list with all points from all sections")
+        .def_property_readonly("section_offsets", [](morphio::Morphology* morpho){
+                return py::array(static_cast<py::ssize_t>(morpho->sectionOffsets().size()), morpho->sectionOffsets().data());
```

Comment:
`Morphology::sectionOffsets` returns a new vector so:
* it should be called only once.
* because the vector is deallocated after the function and because if I remember correctly `py::array` does not copy the data buffer given in parameter, the returned `py::array` points to deleted memory. You may have to use a Python capsule to keep the vector alive as long as the Python object lives and pass a function pointer to Python so that it can destroyed the vector when Python GC destroyed the object. I think @ferdonline has a nice utility function to so that.

Reply:
Can I merge the current version and I'll use @ferdonline function in a seperate commit ?